Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo

```diff
@@ -41,6 +41,35 @@ model RadiatorEN442_2 "Dynamic radiator for space heating"
   parameter Boolean homotopyInitialization = true "= true, use homotopy method"
     annotation(Evaluate=true, Dialog(tab="Advanced"));
 
+  parameter Boolean use_dh = false
```

Comment:
For radiators, it does not make sense to use the hydraulic diameter. Therefore, this parameter should be removed.